examples/advanced/compare_maxcut_parameters.ipynb

```diff
@@ -48,58 +48,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "id": "9b3cead5-3b8c-4d2c-841c-50a0938b1101",
    "metadata": {
     "tags": []
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "QAOA Approximation Ration at p = 1\n",
-      "Fixed: 0.7447449856445802\n",
-      "SK: 0.7446965193396393\n",
-      "QAOA Approximation Ration at p = 2\n",
-      "Fixed: 0.7941748988319051\n",
-      "SK: 0.7909404782710872\n",
-      "QAOA Approximation Ration at p = 3\n",
-      "Fixed: 0.8238045549650369\n",
-      "SK: 0.8206759633288098\n",
-      "QAOA Approximation Ration at p = 4\n",
-      "Fixed: 0.8427741081372286\n",
-      "SK: 0.8394883077961094\n",
-      "QAOA Approximation Ration at p = 5\n",
-      "SK: 0.8555177750871605\n",
-      "QAOA Approximation Ration at p = 6\n",
-      "SK: 0.8683429820234648\n",
-      "QAOA Approximation Ration at p = 7\n",
-      "SK: 0.8783448631130716\n",
-      "QAOA Approximation Ration at p = 8\n",
-      "SK: 0.8862120411609924\n",
-      "QAOA Approximation Ration at p = 9\n",
-      "SK: 0.8925029010830423\n",
-      "QAOA Approximation Ration at p = 10\n",
-      "SK: 0.8977500836882315\n",
-      "QAOA Approximation Ration at p = 11\n",
-      "SK: 0.9022307316009247\n",
-      "QAOA Approximation Ration at p = 12\n",
-      "SK: 0.9060755336973795\n",
-      "QAOA Approximation Ration at p = 13\n",
-      "SK: 0.9093774753834447\n",
-      "QAOA Approximation Ration at p = 14\n",
-      "SK: 0.9122888929726596\n",
-      "QAOA Approximation Ration at p = 15\n",
-      "SK: 0.914928559351209\n",
-      "QAOA Approximation Ration at p = 16\n",
-      "SK: 0.9174003464111532\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Generate a random regular graph\n",
-    "N = 20\n",
+    "N = 10\n",
```

Comment:
Can we put 20 back in? Why switch to 10? 20 should be fast enough with `c` simulator even on CPU.